src/importexport/musicxml/internal/musicxml/musicxmlsupport.cpp

```diff
@@ -588,58 +586,60 @@ QString accSymId2SmuflMxmlString(const SymId id)
 
 SymId mxmlString2accSymId(const QString mxmlName, const QString smufl)
 {
-    QMap<QString, SymId> map;   // map MusicXML accidental name to MuseScore enum SymId
-    map["sharp"] = SymId::accidentalSharp;
-    map["natural"] = SymId::accidentalNatural;
-    map["flat"] = SymId::accidentalFlat;
-    map["double-sharp"] = SymId::accidentalDoubleSharp;
-    map["sharp-sharp"] = SymId::accidentalDoubleSharp;
-    //map["double-flat"] = SymId::accidentalDoubleFlat; // shouldn't harm, but doesn't exist in MusicXML
-    map["flat-flat"] = SymId::accidentalDoubleFlat;
-    map["natural-sharp"] = SymId::accidentalNaturalSharp;
-    map["natural-flat"] = SymId::accidentalNaturalFlat;
-
-    map["quarter-flat"] = SymId::accidentalQuarterToneFlatStein;
-    map["quarter-sharp"] = SymId::accidentalQuarterToneSharpStein;
-    map["three-quarters-flat"] = SymId::accidentalThreeQuarterTonesFlatZimmermann;
-    map["three-quarters-sharp"] = SymId::accidentalThreeQuarterTonesSharpStein;
-
-    map["sharp-down"] = SymId::accidentalQuarterToneSharpArrowDown;
-    map["sharp-up"] = SymId::accidentalThreeQuarterTonesSharpArrowUp;
-    map["natural-down"] = SymId::accidentalQuarterToneFlatNaturalArrowDown;
-    map["natural-up"] = SymId::accidentalQuarterToneSharpNaturalArrowUp;
-    map["flat-down"] = SymId::accidentalThreeQuarterTonesFlatArrowDown;
-    map["flat-up"] = SymId::accidentalQuarterToneFlatArrowUp;
-    map["double-sharp-down"] = SymId::accidentalThreeQuarterTonesSharpArrowDown;
-    map["double-sharp-up"] = SymId::accidentalFiveQuarterTonesSharpArrowUp;
-    map["flat-flat-down"] = SymId::accidentalFiveQuarterTonesFlatArrowDown;
-    map["flat-flat-up"] = SymId::accidentalThreeQuarterTonesFlatArrowUp;
-
-    map["arrow-down"] = SymId::accidentalArrowDown;
-    map["arrow-up"] = SymId::accidentalArrowUp;
-
-    map["triple-sharp"] = SymId::accidentalTripleSharp;
-    map["triple-flat"] = SymId::accidentalTripleFlat;
-
-    map["slash-quarter-sharp"] = SymId::accidentalKucukMucennebSharp;
-    map["slash-sharp"] = SymId::accidentalBuyukMucennebSharp;
-    map["slash-flat"] = SymId::accidentalBakiyeFlat;
-    map["double-slash-flat"] = SymId::accidentalBuyukMucennebFlat;
-
-    map["sharp-1"] = SymId::accidental1CommaSharp;
-    map["sharp-2"] = SymId::accidental2CommaSharp;
-    map["sharp-3"] = SymId::accidental3CommaSharp;
-    map["sharp-5"] = SymId::accidental5CommaSharp;
-    map["flat-1"] = SymId::accidental1CommaFlat;
-    map["flat-2"] = SymId::accidental2CommaFlat;
-    map["flat-3"] = SymId::accidental3CommaFlat;
-    map["flat-4"] = SymId::accidental4CommaFlat;
-
-    map["sori"] = SymId::accidentalSori;
-    map["koron"] = SymId::accidentalKoron;
-
-    if (map.contains(mxmlName)) {
-        return map.value(mxmlName);
+    static std::map<QString, SymId> map;   // map MusicXML accidental name to MuseScore enum SymId
+    if (map.empty()) {
```

Comment:
I would rewrite a lot of things in all this code... which is not done very efficiently. But I don’t want to mix two different tasks... Here I decided to optimize in the simplest way, because it was done too suboptimally.